pkg/api/v1alpha1/tinkerbellmachineconfig.go

```diff
@@ -78,6 +78,18 @@ func GetTinkerbellDefaultTemplateOverride(name string) *TinkerbellTemplate {
 						"CMD_LINE":            "apt -y update && apt -y install openssl",
 					},
 				},
+				{
+					Name:    "create-user",
```

Comment:
I added this task back to the default TinkerbellTemplate as I realized I missed it in the struct PR.